src/bot/eval.py

```diff
@@ -1,25 +1,30 @@
+"""
+This module contains functions
+for evaluating code using the Coliru API.
+"""
+
 import datetime
 import discord
 
 from discord.ext import commands
-from util import coliru
+from util import coliru  # pylint: disable=no-name-in-module
```

Comment:
I don't either.
I tried with a relative import, but I keep getting the following error:
```py
    from .util import coliru
ImportError: attempted relative import with no known parent package
```